### PR TITLE
feat: add deleteByIdBatch interface and my information

### PR DESCRIPTION
**Pull request by shbone**
- add my personal information to `pom.xml`
- add add deleteByIdBatch interface function
- add relative test module

### CODE REVIEW

**Comment by shbone** [comment]
Excuse me! I have a problem. @wey-gu 
I only change several lines of code in `NebulaDaoBasic.java` but there have been 
@@ -1,467 +1,487 @@
I don't know how to fix it.

File: src/main/resources/NebulaDaoBasic.xml [hunk 1]
```diff
@@ -284,6 +284,13 @@
     <delete id="deleteById">
         DELETE VERTEX ${ ng.valueFmt( p0 ) }
     </delete>
+
+
+    <delete id="deleteByIdBatch">
+        @for ( v in ng_args[0] ) {
+        DELETE VERTEX ${ ng.valueFmt( v )};
```
Comment (CorvusYe) [inline src/main/resources/NebulaDaoBasic.xml:291]: The use of passing in multiple ids at the same time may be more appropriate than the use of multiple statement structures.
https://docs.nebula-graph.com.cn/3.6.0/3.ngql-guide/12.vertex-statements/4.delete-vertex/
```cql
DELETE VERTEX <vid> [ , <vid> ... ] [WITH EDGE];
```

Comment (shbone) [reply src/main/resources/NebulaDaoBasic.xml:291]: OK, I learn that and will change it later!

**Review by CorvusYe — APPROVED** [review]
Thank you for your efforts and waiting.
That's very elegant.
We can merge without any additional changes.
cc: @wey-gu

**Comment by CorvusYe** [comment]
@shbone 
I need to trouble you again to handle the conflict.
Thanks !

**Comment by shbone** [comment]
> @shbone I need to trouble you again to handle the conflict. And add some infomations in this sections.
> 
> https://github.com/nebula-contrib/ngbatis/blob/c67c017fc1cb28718c7580cd86f72f50f684e670/CHANGELOG.md?plain=1#L29-L44
> 
> Two PRs should belong to `## Features`
> 
> Your efforts should be perceived by people.
> 
> Thank you vvvvvery much!

Thank you ! I will  fix this conflict and edit the `[ngbatis/CHANGELOG.md]`

**Comment by shbone** [comment]
> @shbone I need to trouble you again to handle the conflict. And add some infomations in this sections.
> 
> https://github.com/nebula-contrib/ngbatis/blob/c67c017fc1cb28718c7580cd86f72f50f684e670/CHANGELOG.md?plain=1#L29-L44
> 
> Two PRs should belong to `## Features`
> 
> Your efforts should be perceived by people.
> 
> Thank you vvvvvery much!

And I am sorry that  I don't know why I have this conflict between `src/main/java/org/nebula/contrib/ngbatis/proxy/NebulaDaoBasic.java` and `
src/main/resources/NebulaDaoBasic.xml`. It is hard for me, whether I should delete the `Param` and change it to `List<>` ? Thank you!!

**Comment by CorvusYe** [comment]
This might have something to do with #234 using the master branch and now using another branch?
You can try merging the current master branch into the local.

**Comment by wey-gu** [comment]
Let me give a try on resolving it.

**Comment by wey-gu** [comment]
@shbone  Could you double-check check I didn't miss any changes of yours here?

if it looks good, we are good to merge.

I guess the conflict was related to the new line character introduced in a windows env.

**Comment by shbone** [comment]
Yes, I have checked it and all is right

**Comment by wey-gu** [comment]
Let's merge then! :D

**Pull request merged**